### DISCUSSION TITLE
Feature "object with additionalProperties" is not implemented yet.

### DISCUSSION THREAD

**Discussion by mattoni** [ogen-go/ogen#894] [Q&A] [answered]
I just tried generating our schema with ogen, and got `Feature "object with additionalProperties" is not implemented yet.`. I don't see anywhere in the docs that additionalProperties ISN'T supported, so I wanted to clarify that this is intended, and if so is there a plan to fix it?

Thanks!

**Comment by ghost** [comment, marked as answer]
Currently, `ogen` does not support `additionalProperties` in parameter defintions, mostly because OpenAPI specification does not clarify what fields exactly should be considered as _additional_.

Sorry for late response. Didn't get a notification for some reason.
